### ISSUE TITLE
Batch normalization update ops.

### ISSUE THREAD

**Issue by Gouiaa** [Stick-To/DLA-tensorflow#3]
https://github.com/Stick-To/DLA-tensorflow/blob/9360c90d29e89929c6c48317971a51547685fb7a/DLA.py#L99

@Stick-To I think you forgot to update the batch normalization  `moving_variance` and `moving_mean`.  So, I think you have to add this line before the optimizer:

```
update_ops = tf.get_collection(tf.GraphKeys.UPDATE_OPS)
with tf.control_dependencies(update_ops):
    train_op = optimizer.minimize(total_loss)
```

**Comment by Stick-To** [comment]
thanks, it is fixed now
